plugins/azure/databricks/workspaceDbfsInfraEncryption.js

```diff
@@ -0,0 +1,54 @@
+var async = require('async');
+var helpers = require('../../../helpers/azure');
+
+module.exports = {
+    title: 'Databricks Workspace DBFS Infrastructure Encryption',
+    category: 'AI & ML',
+    domain: 'Machine Learning',
+    severity: 'Medium',
+    description: 'Ensures that DBFS root for Databricks workspace has Infrastrucure encryption enabled.',
```

Comment:
```suggestion
    description: 'Ensures that DBFS root for Databricks workspace has infrastrucure encryption enabled.',
```
can we also specify that this is for premium workspaces only? also can the DBFS root parts everywhere be phrased a little better like 'Ensures that Databricks workspace has infrastrucure encryption enabled for DBFS root directory'